lxc/remote.go

```diff
@@ -55,6 +56,35 @@ func (c *remoteCmd) flags() {
 	gnuflag.BoolVar(&c.public, "public", false, i18n.G("Public image server"))
 }
 
+func getRemoteCertificate(address string) (*x509.Certificate, error) {
+	// Setup a permissive TLS config
+	tlsConfig, err := shared.GetTLSConfig("", "", nil)
+	if err != nil {
+		return nil, err
+	}
+
+	tlsConfig.InsecureSkipVerify = true
```

Comment:
This is a pretty big red flag. Reading through where this is called, maybe its ok, but you should probably put a big comment indicating why it is ok to not validate the remote certificate is signed by someone you trust.